package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-runner",
-  "version": "0.3.9",
+  "version": "0.4.0",
```

Comment:
This version isn't incompatible with previous ones, so maybe release it as 0.3.10 instead?